### PR TITLE
Add VB example to Xaml designer extensibility

### PR DESCRIPTION
**Pull request by Nukepayload2**
Issue: #3602

Before creating your pull request, please check your content against these quality criteria:

- Did you consider search engine optimization (SEO) when you chose the title in the metadata section and the H1 heading (i.e. the displayed title that starts with a single #)?
- For new articles, did you add it to the table of contents?
- Did you update the "ms.date" metadata for new or significantly updated articles?
- Are technical terms and concepts introduced and explained, and are acronyms spelled out on first mention?
- Should this page be linked to from other pages or Microsoft web sites?

For more information about creating content for docs.microsoft.com, see the contributor guide at https://docs.microsoft.com/contribute/.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@Nukepayload2 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/extensibility/xaml-designer-extensibility-migration.md [hunk 1]
```diff
@@ -80,6 +104,16 @@ if (type != null && buttonType != type.IsSubclassOf(buttonType))
 }
```
Comment (mgoertz-msft) [inline docs/extensibility/xaml-designer-extensibility-migration.md]: Would you mind fixing the C# sample too? I'm pretty sure this was meant to say this:
`if (type != null && type.IsSubclassOf(buttonType))` or shorter to match your change:
`if (type?.IsSubclassOf(buttonType) == true)`

**Comment by lutzroeder** [comment]
#sign-off

**Comment by PRMerger10** [comment]
Only Microsoft employees can sign off on PRs. Your PR will be routed for review.

**Comment by Jak-MS** [comment]
@lutzroeder - I'll merge this, but I can't verify that the links in line 44 are working correctly. Can you check them after the changes are live? Thanks.

**Pull request merged**